build-configs/configs/integreat-test-cms/index.js

```diff
@@ -6,7 +6,10 @@ import type { BuildConfigType } from '../BuildConfigType'
 const IntegreatTestCmsBuildConfig: BuildConfigType = {
   ...IntegreatBuildConfig,
   cmsUrl: 'https://cms-test.integreat-app.de',
-  switchCmsUrl: 'https://cms.integreat-app.de'
+  switchCmsUrl: 'https://cms.integreat-app.de',
+  iosBuildOptions: {
```

Comment:
Not sure about naming here. Any ideas?

Reply:
I don't see how this is useful. Why is `BUILD_CONFIG_APP_NAME` different to `integreat-test-cms` (build-config-id) AND `Integreat` (appName)? What kind of app name is it then?

If it's because we want to show the dev different names, I'd prefer simply switching the `appName` in the `integreat-test-cms` BuildConfig to `IntegreatTestCms`.

Reply:
I'd go with only `ios` or `native-ios`. It is already clear that we deal with build options as `switchCmsUrl` is also a Build Option, right?

Reply:
It should not be different, just forgot to change the `appName` here.

My reason for a separate key is the generation of xcconfig files. If we just map each key (e.g. `appName`, `lightTheme`, ...) to its value, the xcconfig files are probably not valid anymore. Currently therefore only `iosBuildOptions` keys are included in the xcconfig files.

One idea would possible to just include a build config option only in the xcconfig file if the value is of type string. Any other suggestions?

Reply:
Will be flattened and uppercased :)